app/components/Feedback/index.tsx

```diff
@@ -39,48 +40,59 @@ const Feedback = ({
   upHint,
   downHint,
   options,
+  formClassName,
   onSubmit,
 }: FeedbackProps) => {
-  const [showFeedback, setShowFeedback] = useState(false)
+  const [showThanks, setShowThanks] = useState(false)
   const [showFeedbackForm, setShowFeedbackForm] = useState(false)
+  const [voted, setVoted]: any = useState(false)
 
   useEffect(() => {
-    const timeout = setInterval(() => setShowFeedback(false), 6000)
+    const timeout = setTimeout(() => setShowThanks(false), 6000)
     return () => clearInterval(timeout)
-  }, [showFeedback])
+  }, [showThanks])
 
   return (
-    <div className="feedback relative">
-      <CompositeButton className="flex-container">
+    <div className="feedback">
+      <ButtonSecondaryWrapper>
         <Action
           pageid={pageid}
           showText={!!labels}
           actionType={ActionType.HELPFUL}
+          disabled={voted}
           hint={upHint}
-          onClick={() => setShowFeedback(true)}
+          setVoted={setVoted}
+          onClick={() => {
+            setShowThanks(true)
+          }}
         />
         <Action
           pageid={pageid}
           showText={!!labels}
           hint={downHint}
           actionType={ActionType.UNHELPFUL}
+          disabled={voted}
+          setVoted={setVoted}
           onClick={() => {
-            setShowFeedback(!showForm)
+            if (!showForm) setShowThanks(true)
             setShowFeedbackForm(!!showForm)
           }}
         />
-      </CompositeButton>
+      </ButtonSecondaryWrapper>
 
-      {showFeedback && <div className="thanks">Thanks for your feedback!</div>}
+      <p className={'thanks ' + (showThanks ? 'show' : 'hide')}>Thank you for your feedback!</p>
 
       {showFeedbackForm && (
         <FeedbackForm
-          onSubmit={onSubmit}
+          onSubmit={(att) => {
```

Comment:
can you name this something more intuitive?

Reply:
Too true